openquake/mbt/tests/tools/faults_test.py

```diff
@@ -25,13 +25,16 @@ def testcase01(self):
         m_upp = 7.5
         b_gr = 1.1
         bin_width = 0.05
+        # to pass test
+        m_min = 6.5
 
         # Compute moment in Nm
         expected = (32 * 1e9) * (area * 1e6) * (slip_rate * 1e-3)
 
         # Compute rates and magnitudes (i.e. bin centers)
-        rates = rates_for_double_truncated_mfd(area, slip_rate, m_low, m_upp,
+        rates = rates_for_double_truncated_mfd(area, slip_rate, m_low, m_upp, m_min,
```

Comment:
I would have preferred to keep this function unchanged and select the rates a-posteriori.

Reply:
I think this is not necessary, but, yes,  it is possible